ros_controllers/test/test_trajectory_controller.cpp

```diff
@@ -195,47 +195,47 @@ TEST_F(TestTrajectoryController, configuration) {
   executor.cancel();
 }
 
-TEST_F(TestTrajectoryController, activation) {
-  auto traj_controller = std::make_shared<ros_controllers::JointTrajectoryController>(
-    joint_names, op_mode);
-  auto ret = traj_controller->init(test_robot, controller_name);
-  if (ret != controller_interface::CONTROLLER_INTERFACE_RET_SUCCESS) {
-    FAIL();
-  }
-
-  auto traj_lifecycle_node = traj_controller->get_lifecycle_node();
-  rclcpp::executors::MultiThreadedExecutor executor;
-  executor.add_node(traj_lifecycle_node->get_node_base_interface());
-
-  auto state = traj_lifecycle_node->configure();
-  ASSERT_EQ(state.id(), State::PRIMARY_STATE_INACTIVE);
-
-  state = traj_lifecycle_node->activate();
-  ASSERT_EQ(state.id(), State::PRIMARY_STATE_ACTIVE);
-
-  // wait for the subscriber and publisher to completely setup
-  std::this_thread::sleep_for(std::chrono::seconds(1));
-
-  // send msg
-  builtin_interfaces::msg::Duration time_from_start;
-  time_from_start.sec = 1;
-  time_from_start.nanosec = 0;
-  std::vector<std::array<double, 3>> points {{{3.3, 4.4, 5.5}}};
-  publish(time_from_start, points);
-  // wait for msg is be published to the system
-  std::this_thread::sleep_for(std::chrono::milliseconds(500));
-  executor.spin_once();
-
-  traj_controller->update();
-  test_robot->write();
-
-  // change in hw position
-  EXPECT_EQ(3.3, test_robot->pos1);
-  EXPECT_EQ(4.4, test_robot->pos2);
-  EXPECT_EQ(5.5, test_robot->pos3);
-
-  executor.cancel();
-}
+// TEST_F(TestTrajectoryController, activation) {
```

Comment:
Any reason for removing this test?

Reply:
see #4

Reply:
I saw the [TODO](https://github.com/ros-controls/ros2_controllers/blob/4322e8d0031f71d6e207fa8de5a48bb189615684/ros_controllers/src/trajectory.cpp#L27). I can create a PR and update when this lands then.